### ISSUE TITLE
FAIL: runtests

### ISSUE THREAD

**Issue by andreas-schwab** [libexpat/libexpat#827]
FAIL: runtests
==============

taken=53156 baseline=3300 slowdown=16 max_slowdown=15
ERROR: Compared to the baseline runtime of the first test, this test has a slowdown of more than <max_slowdown>. Please keep increasing the value by 1 until it reliably passes the test on your hardware and open a bug sharing that number with us. Thanks in advance!
Expat version: expat_2.6.0
FAIL [chunksize=0 deferral=1]: test_big_tokens_take_linear_time (max_slowdown=15 text="<e><aaaaaa/></e>" at basic_tests.c:5300)
99%: Checks: 4320, Failed: 1
FAIL runtests (exit status: 1)

**Comment by hartwork** [comment]
Hi @andreas-schwab, sorry for the inconvenience. I believe this is fixed by #817 (and a duplicate of #826). I am wondering two things now:
- how much trouble does/will the issue cause to you in practice?
- is applying pull request #817 as a patch on top an option in your environment?

**Comment by andreas-schwab** [comment]
basic_tests.c: In function ‘test_big_tokens_scale_linearly’:
basic_tests.c:5237:5: error: ‘g_bytesScanned’ undeclared (first use in this function)
 5237 |     g_bytesScanned = 0;
      |     ^~~~~~~~~~~~~~
basic_tests.c:5237:5: note: each undeclared identifier is reported only once for each function it appears in
At top level:
cc1: note: unrecognized command-line option ‘-Wno-pedantic-ms-format’ may have been intended to silence earlier diagnostics
make[3]: *** [Makefile:821: basic_tests.o] Error 1
make[3]: Leaving directory '/home/abuild/rpmbuild/BUILD/expat-2.6.0/tests'

**Comment by andreas-schwab** [comment]
Together with fe0177cd it builds now.

**Comment by hartwork** [comment]
@andreas-schwab that makes sense, the pull request is two commits and fe0177cd3fe73ab2ad2e5e749a6badbbd7ec1d83 is needed to make the follow-up dc8499f295bcb5e0b1ff89aa3ca5ca65ddb4ca7b compile.

May I ask which distro you are building on or packaging for?

CC @kraptor (packaging 2.6.0 for SUSE)

**Comment by andreas-schwab** [comment]
https://build.opensuse.org/package/show/openSUSE:Factory:RISCV/expat

**Comment by hartwork** [comment]
@andreas-schwab thanks for the link!

I will close the issue as fixed due to #817 being merged to master.
